templates/security/certifications.html

```diff
@@ -3,202 +3,202 @@
-      </p>
-    </div>
-    <div class="col-4 u-align--center u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/961a1ad1-csec-logo-removebg-preview.png?w=150" width="150" alt="">
-    </div>
-  </div>
+<section class="p-strip" id="compliance-profiles">
+ <div class="u-fixed-width">
+   <h2>Ubuntu compliance profiles</h2>
+   <p class="u-sv2">The default configuration of Ubuntu LTS releases, as provided by Canonical, balances between usability, performance and security. However, non general purpose systems can be further hardened to reduce their attack surface. Reducing the attack surface is often part of the compliance with the organization’s cybersecurity framework, but is also a widely accepted security best practice. We recommend using the industry accepted benchmarks below. Click on each benchmark for more detailed information.</p>
+   <table class="p-table--compliance-profiles">
+     <thead>
+       <th class="u-hide--small"></th>
+       <th></th>
+       <th>16.04</th>
+       <th>18.04</th>
+       <th>20.04</th>
+     </thead>
+     <tbody>
+       <tr>
+         <td class="u-hide--small">
+         {{ image (
+           url="https://assets.ubuntu.com/v1/f98af83d-cis-logo-removebg-preview.png",
+           alt="CIS logo",
+           width="50",
+           height="50",
+           hi_def=True,
+           loading="lazy"
+           ) | safe
+         }}
+         </td>
+         <td>
+           <a href="/security/cis">Center for Internet Security (CIS) certified benchmarks for Ubuntu systems</a>
+         </td>
+         <td><i class="p-icon--success">Yes: Tooling and automation</i></td>
+         <td><i class="p-icon--success">Yes: Tooling and automation</i></td>
+         <td><i class="p-icon--success">Yes: Tooling and automation</i></td>
+       </tr>
+       <tr>
+         <td class="u-hide--small">
+         {{ image (
+           url="https://assets.ubuntu.com/v1/ef01809f-DISA-logo-transparent.png",
+           alt="DISA logo",
+           width="136",
+           height="50",
+           hi_def=True,
+           attrs={"style": "vertical-align: middle;"},
+           loading="lazy"
+           ) | safe
+         }}
+         </td>
+         <td>
+           <a href="/security/disa-stig">Defence Information System Agency (DISA) Security Technical Implementation Guides (STIGs)</a>
+         </td>
+         <td>{{ image(url="https://assets.ubuntu.com/v1/2ccda8d7-tick-orange.svg", alt="Yes: STIG content", width="14", height="14", hi_def=True, loading="lazy",) | safe }}</td>
```

Comment:
We have a mix of ticks here

Reply:
👍🏼  I've swapped out the green tick icons for assets